RNGoogleMobileAdsExample/App.tsx

```diff
@@ -1025,7 +1030,7 @@ TestRegistry.registerTest(new DebugMenuTest());
 const App = () => {
   return (
     <SafeAreaView>
-      <ScrollView contentInsetAdjustmentBehavior="automatic">
+      <ScrollView contentContainerStyle={{ paddingRight: 15 }} contentInsetAdjustmentBehavior="automatic">
```

Comment:
This was a quick hack because there are too many tests to fit on a screen already and they're all 100% pressable area, so you can't scroll the scroll view. This gives you some space on the right to scroll it.